addon/-private/serializers/embedded-records-mixin.js

```diff
@@ -321,16 +372,38 @@ export default Ember.Mixin.create({
       this._super(snapshot, json, relationship);
       return;
     }
-    var includeIds = this.hasSerializeIdsOption(attr);
-    var includeRecords = this.hasSerializeRecordsOption(attr);
-    if (includeIds) {
+
+    if (this.hasSerializeIdsOption(attr)) {
       let serializedKey = this.keyForRelationship(attr, relationship.kind, 'serialize');
       json[serializedKey] = snapshot.hasMany(attr, { ids: true });
-    } else if (includeRecords) {
+    } else if (this.hasSerializeRecordsOption(attr)) {
       this._serializeEmbeddedHasMany(snapshot, json, relationship);
+    } else if (isEnabled("ds-serialize-ids-and-types") && this.hasSerializeIdsAndTypesOption(attr)) {
```

Comment:
@mmun would you accept a PR for [`babel-plugin-feature-flags`](https://github.com/ember-cli/babel-plugin-feature-flags) which replaces calls to `isEnabled("xxx")` with either `true` or `false` if a specific option is passed to the babel plugin?